### DISCUSSION TITLE
Regular Failures

### DISCUSSION THREAD

**Discussion by Bag72** [alexjustesen/speedtest-tracker#1319] [Q&A]
I've been having regular failures on the checking.  The results page show started and then failed. However I've been using the system to download files and browse the web and have not noticed any gaps or freezes.   The system is slow as I'm hammering it atm with compressing videos and working on a web site development.  Could it be I'm causing a timeout?

Is there any way I can check what is going on?    Can I look at any logs or run a cli script or something that would give me an idea of what the issue is?

**Comment by lnlyssg** [comment]
I'd also be keen to know this, I've had a number of failures today and I'm not sure if it's a "bad" server or an issue at my end

**Comment by Bag72** [comment]
I was playing with it earlier as I'm convinced my link has not gone down.  I has not disconnected a ssh I've had going since yesterday morning.  I wondered if it is something to do with the Speedtest Servers I used.   When I started I limited them to local only and I forgot to clear them when I went live.   So this morning I cleared the list and, fingers crossed, I've had no errors since.  I'll keep an eye on it.

**Reply by Bag72** [reply]
Update: Still getting errors but appears to be much less.  Still have no idea what they are as the logs don't give any information on them.  I suspect it can only be, and I'm happy to be corrected, that some of the servers it selects have issues.
I'll keep tracking.
